tests/cpp/aot/llvm/mpm88_graph_aot_cpu.py

```diff
@@ -0,0 +1,10 @@
+import os
+import sys
+
+import taichi as ti
+
+sys.path.append(os.path.join(os.path.dirname(os.path.abspath(__file__)), ".."))
```

Comment:
nit: I'm personally not a big fan of manipulating `sys.path`. I'm not too worried about this particular use case but in general afraid of changing `sys.path` spreading across the codebase. 
Can we try passing in arch information to the script?

Reply:
Thanks! Switched to pass "PYTHONPATH" as environmental variables during subprocess call (at tests/run_tests.py).

Reply:
Thanks! Switched to pass in arguments~